log/securedrop-log-saver

```diff
@@ -34,7 +34,7 @@ def main():
                 except OSError as err:
                     if err.errno != errno.EEXIST:
                         raise
-                fh = open(filepath, "a")
+                fh = open(filepath, "a")  # noqa: SIM115
```

Comment:
(in case anyone's curious, https://docs.astral.sh/ruff/rules/open-file-with-context-handler/)